source/index.txt

```diff
@@ -19,46 +19,83 @@ MongoDB Go Driver
 Introduction
 ------------
 
-The official MongoDB Go Driver enables Go applications to connect to
-MongoDB and work with data.
+The official MongoDB Go Driver enables Go applications to connect to and
+work with data in MongoDB. This page describes each section of the
+driver documentation. 
+
+Learn
+-----
+
+Visit the Developer Hub for content, resources, community, and social
+engagement for developers related to the Go driver:
+
+- To view articles on Go, see the `How To's and Articles page <https://www.mongodb.com/developer/learn/?content=Articles&text=Go>`__.
```

Comment:
Question: why is this called How To's? I just see blog style posts and article, videos, podcasts at the bottom.

Reply:
It's the actual name of the tab when you hover over "Learn" at the top bar of the page